### DISCUSSION TITLE
Giving downloader access to a private repository

### DISCUSSION THREAD

**Discussion by ghost** [oss-review-toolkit/ort#7074] [General]
Hi, 

I'm currently attempting to run the downloader on a project that sources some of its dependencies from private git repositories. Is there anyway I can configure the downloader in a way that will allow me to access those for scanning?

**Comment by MarcelBochtler** [comment]
Hi @kmq2022,

yes ORT allows multiple ways for authentication.
In your case you might need a `.netrc` with the credentials. See [this documentation](https://github.com/oss-review-toolkit/ort/blob/013e9d56f0e563d166bcd2b2d52b632a87cd056b/docs/hints-for-use-with-docker.md#authenticating-with-a-private-git-repository-fails) how to configure it.
